fdbclient/FDBTypes.h

```diff
@@ -974,6 +974,22 @@ struct HealthMetrics {
 	}
 };
 
+struct DDMetrics {
```

Comment:
By convention this should probably be renamed to DDMetricsRef, since it does not own its memory

Reply:
Renamed to DDMetricsRef.